ddtrace/settings/_otel_remapper.py

```diff
@@ -0,0 +1,190 @@
+    return new_value
+
+
+def _remap_traces_sampler(otel_value):
+    """Remaps the otel trace sampler to ddtrace trace sampler"""
+    if otel_value == "always_on" or otel_value == "parentbased_always_on":
+        new_value = "1.0"
+    elif otel_value == "always_off" or otel_value == "parentbased_always_off":
+        new_value = "0.0"
+    elif otel_value == "traceidratio" or otel_value == "parentbased_traceidratio":
+        new_value = os.environ.get("OTEL_TRACES_SAMPLER_ARG", "1")
+    return new_value
+
+
+def _remap_traces_exporter(otel_value):
+    """Remaps the otel trace exporter to ddtrace trace exporter"""
+    if otel_value != "none":
+        log.warning("An unrecognized trace exporter '%s' is being used; setting dd_trace_enabled to false.", otel_value)
+        new_value = "False"
+    else:
+        new_value = "True"
+    return new_value
+
+
+def _remap_metrics_exporter(otel_value):
+    """Remaps the otel metrics exporter to ddtrace metrics exporter"""
+    if otel_value != "none":
+        log.warning(
+            "An unrecognized runtime metrics exporter '%s' is being "
+            "used; setting dd_runtime_metrics_enabled to false.",
+            otel_value,
+        )
+    new_value = "False"
+    return new_value
+
+
+def _remap_logs_exporter(otel_value):
+    """Remaps the otel logs exporter to ddtrace logs exporter"""
+    if otel_value != "none":
+        log.warning("Unsupported logs exporter detected.")
+        new_dd_value = ""
+        return new_dd_value
+    return otel_value
+
+
+def _remap_otel_tags(otel_value):
+    """Remaps the otel tags to ddtrace tags"""
+    dd_tags = []
+    otel_tags = otel_value.split(",")
+    otel_user_tag_dict = dict()
+
+    for tag in otel_tags:
+        tag_pair = tag.split("=")
+        otel_user_tag_dict[tag_pair[0]] = tag_pair[1]
+
+    for otel_key, dd_key in OTEL_UNIFIED_TAG_MAPPINGS.items():
+        if otel_key in otel_user_tag_dict.keys():
+            dd_tags.append("{}:{}".format(dd_key, otel_user_tag_dict[otel_key]))
+
+    for key, value in otel_user_tag_dict.items():
+        if len(dd_tags) < 11 and key not in [
+            "deployment.environment",
+            "service.name",
+            "service.version",
+        ]:
+            dd_tags.append("{}:{}".format(key, value))
+
+    if len(otel_user_tag_dict.items()) > 10:
+        ten_tags = dd_tags[:10]
+        log.warning(
+            "To preserve metrics cardinality, only the following first 10 tags have been processed %s",
+            ten_tags,
+        )
+    new_value = ",".join(dd_tags)
+    return new_value
+
+
+def _remap_otel_sdk_config(otel_value):
+    """Remaps the otel sdk config to ddtrace sdk config"""
+    if otel_value == "false":
+        new_value = "True"
+    elif otel_value == "true":
+        new_value = "False"
+    else:
+        log.warning("Unexpected value for OTEL_SDK_DISABLED.")
+        return otel_value
```

Comment:
From their docs it seems like it strictly takes true or false but will throw in the support for 0 or 1 just in case!:

> Boolean value. If “true”, a no-op SDK implementation will be used for all telemetry signals. Any other value or absence of the variable will have no effect and the SDK will remain enabled. This setting has no effect on propagators configured through the OTEL_PROPAGATORS variable